prompting/apparmor/apparmor.go

```diff
@@ -0,0 +1,19 @@
+// Package apparmor implements high-level interface to a subset of AppArmor features
+package apparmor
+
+import (
+	"path/filepath"
+
+	"github.com/snapcore/snapd/dirs"
+	"github.com/snapcore/snapd/osutil"
+)
+
+const notifyPath = "/sys/kernel/security/apparmor/.notify"
+
+func PromptingAvailable() bool {
+	return osutil.FileExists(notifyPath)
+}
+
+func NotifyPath() string {
```

Comment:
it's a bit outside of our style for this to be a function, see for example how sandbox/apparmor.ConfDir etc are managed

Reply:
Fixed, thanks!